sources/playground/default_client.c

```diff
@@ -28,18 +19,17 @@ static int sqlite_trace_callback(unsigned type, void* ctx, void* p, void* x) {
 
 // patternlint-disable-next-line prefer-sized-ints-in-msys
 int main(int argc, char **argv) {
-  printf("CQL Mini App Thingy\n");
-
   sqlite3 *db = NULL;
-  SQL_E(sqlite3_open(":memory:", &db));
```

Comment:
This can fail.

Reply:
Another interesting option btw is to specify the db name because persisting some of the databases also makes an interesting demo.

Reply:
I simplified the code for clarity, especially for those less familiar with C, assuming the in-memory SQLite DB would be reliable. I'll reintroduce error handling to offer a better foundation for real-world implementations.

Adding an option to provide a path for the database is a great idea. I'll make it default to ":memory:".